packages/app/pages/AllProjectsPage/AllProjectsPage.tsx

```diff
@@ -26,7 +26,7 @@ interface Props {
  */
 class NavBar extends React.Component {
 
-  readonly navButtons = ["All", "Cooperative", "Projects", "Organizations", "Groups"]
+  readonly navButtons = ["All"]
```

Comment:
Missing semi-colon

Reply:
Why is "All" included? If there are no other ways to filter, the single selector seems out of place. It doesn't do anything for the UI as opposed to just having them all listed with no selectors.

Reply:
It was a justification for leaving the selector code in so we could easily add filters later, since it does not benefit the UI though I will hide it.

Reply:
That makes sense. Yeah I would leave a stub in there since it will likely be implemented in the future, but have it not show on the UI for now.